pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -779,7 +783,7 @@ func getEngine() *wfengine.WorkflowEngine {
 	})
 
 	if err := actors.Init(); err != nil {
-		panic(err)
+		assert.FailNow(t, "failed to init actors: %s", err)
```

Comment:
It's better to use `assert.NoError(t, err)` for cases like this.

Reply:
Or actually, `require.NoError` in this case. The difference is that `require` aborts the current test.

Reply:
> Or actually, `require.NoError` in this case. The difference is that `require` aborts the current test.

Oh wow, I didn't know about this. I have a lot of test code that I need to go back and clean up now. 😅